measure/src/main/kotlin/com/alliander/open/measure/Measure.kt

```diff
@@ -89,4 +89,32 @@ data class Measure<U : Units>(val amount: BigDecimal, val units: U) : Comparable
     override fun toString(): String {
         return "$amount$units"
     }
+
+    fun isZero(): Boolean = this.amount.signum() == 0
+
+    fun isNegative(): Boolean = this.amount < BigDecimal.ZERO
```

Comment:
isPositive lijkt mij dan ook nog we een mooie toevoeging, toch? :D

Reply:
Is het misschien beter om dan sign() toe te voegen ipv isZero, isNegative en isPositive?

Reply:
Ik zou ze alle vier toevoegen, waarbij de `isPositive`/`isZero`/`isNegative` gebruikt maakt van de `sign` functie. Ik vind die syntactic sugar is wel lekker expressief.

Reply:
Ik zal ze alle vier toe voegen voor de duidelijkheid